### DISCUSSION TITLE
Capstone project - Plot Doesn't one to be in one single page

### DISCUSSION THREAD

**Discussion by MatthewBradleyLo** [teamalgoritma/community#714] [Q&A] [answered]
Saya ada issue dimana plot1 saya itu tidak mau stay di halaman pertama doang tapi dia malahan ikut ke halaman yang lain. Cara untuk seperatenya gimana ya?`

![Screenshot (6)](https://github.com/teamalgoritma/community/assets/133225102/a17306ae-b845-45cc-8e26-7a878f339a19)
![Screenshot (5)](https://github.com/teamalgoritma/community/assets/133225102/1cea5388-f7cb-455a-b2e6-b39f4386643e)

**Comment by ranydwi** [comment, marked as answer]
Dear @MatthewBradleyLo 

Plot yang muncul berulang pada beberapa halaman tersebut disebabkan Bapak memiliki nama `tabName` dan juga `inputId` yang sama antara plot pada halaman kedua dan ketiga. Pastikan untuk nama-nama tersebut memiliki nama yang unique ya Pak.

Semoga membantu.
Regards.
